core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -201,6 +201,13 @@ private[spark] object Utils extends Logging {
     // scalastyle:on classforname
   }
 
+  // scalastyle:off classforname
+  /** Preferred alternative to Class.forName(className, initialize, classloader) */
+  def classForName(className: String, initialize: Boolean): Class[_] = {
```

Comment:
Just modify the existing method to take an optional param?

Reply:
This change is no longer relevant, but I still see the benefit of doing this - though for now it just helps to get rid of unnecessary scalastyle off and on in Utils. Will raise a new MINOR PR to isolate two issues.